tests/test_multiple_repositories_integration.py

```diff
@@ -134,20 +126,18 @@ def setUp(self):
 
     # Creates a subprocess running server and uses temp file for logging.
```

Comment:
I have to change all occurrences of `temp file` in the comments when creating the server subprocess...